index.html

```diff
@@ -289,14 +289,14 @@ <h5 class="title-recent-works-one">Multi-Post Stories</h5>
               <input type="email" name="email_address" id="email-address" placeholder="Email address" required>
```

Comment:
- [ ] **[OPTIONAL]**

- Please, kindly fix this issue of width with your inputs
![input](https://user-images.githubusercontent.com/59397159/165435823-8c01cff3-e8c7-4400-8624-418745e39b20.png)

Reply:
@zieeco Thank you for the review, I will fix that now! Excellent view.